### DISCUSSION TITLE
Page scrolls to top when selecting text style drop down

### DISCUSSION THREAD

**Discussion by jarmitage** [knowledgefutures/pubpub#1727] [Support]
As the title suggests, I have been experiencing a bug for a while now, where the page scrolls to top when selecting text style drop down.

Chrome Version 96.0.4664.110 (Official Build) (x86_64)

Video demonstration attached

https://user-images.githubusercontent.com/2885827/147935722-c7237665-9f8b-43d6-82bd-e15eb07c2e26.mov

**Comment by jarmitage** [comment]
In fact, this behaviour occurs whenever I use the toolbar

**Comment by gabestein** [comment]
Hi Jack — unfortunately, I'm unable to replicate this behavior on that version of Chrome. Do you have any Chrome plugins installed that might be conflicting? In particular, PubPub struggled with plugins that attempt to take over or enrich parts of the dom within our text editor.

**Reply by jarmitage** [reply]
Thanks for investigating.

I did check this in an incognito tab and found the same behaviour.

Tomorrow I will try again with the dev tools display up and see if there's anything interesting there.

**Reply by gabestein** [reply]
Thanks, and sorry you're dealing with the issue. Do let us know if you see anything, and feel free to post a screenshot of the console.

**Reply by jarmitage** [reply]
Ok, more digging. Bug behaviour still happening in Chrome Incognito. I have not been able to repro in Safari or Firefox.

When clicking on anything in the toolbar the following warning appears in Chrome Dev Console (same in Safari and Firefox):

```js
`preventOverflow` modifier is required by `hide` modifier in order to work, be sure to include it before `hide`!
(anonymous) @ instrument.js:110
J @ popper.js:1392
fn @ popper.js:1967
(anonymous) @ popper.js:900
H @ popper.js:887
V @ popper.js:948
value @ popper.js:2544
(anonymous) @ popper.js:46
Promise.then (async)
(anonymous) @ popper.js:44
e @ popper.js:2526
(anonymous) @ PopoverState.js:76
os @ react-dom.production.min.js:211
is @ react-dom.production.min.js:212
hc @ react-dom.production.min.js:255
t.unstable_runWithPriority @ scheduler.production.min.js:19
Uo @ react-dom.production.min.js:122
pc @ react-dom.production.min.js:248
Zs @ react-dom.production.min.js:239
(anonymous) @ react-dom.production.min.js:123
t.unstable_runWithPriority @ scheduler.production.min.js:19
Uo @ react-dom.production.min.js:122
Ko @ react-dom.production.min.js:123
Go @ react-dom.production.min.js:122
$s @ react-dom.production.min.js:230
va @ react-dom.production.min.js:163
(anonymous) @ HiddenState.js:67
r @ Dialog.js:341
r @ helpers.js:71
```

Now, interestingly, when I am in Dev Console and click Toggle Device Toolbar, the warning still appears, however the buggy behaviour stops.

**Reply by jarmitage** [reply]
@gabestein any further clues? Sadly I am still having this issue in Chrome, even in Incognito.

**Reply by gabestein** [reply]
Unfortunately, we're unable to replicate the behavior you're seeing. That error is unrelated.

Is it possible you have an extension installed (many function in incognito) that could be causing this behavior?

**Reply by jarmitage** [reply]
I double checked that I don't have any extensions running in Incognito. 

I also just now created a Chrome Profile from scratch and reproduced the behaviour there too.

Bizarre!

Maybe a completely fresh install of Chrome next?

**Reply by gabestein** [reply]
Yup, this is very odd. I really can't think what would be causing this. You could try updating Chrome (or a fresh install, but that seems extreme). Or, I guess one thing I haven't been considering, is it all Pubs, or just this one?

**Reply by jarmitage** [reply]
Confirmed that it happens in other Pubs.

I am willing to nuke my Chrome over this since it seems like there might be issues with it anyway. Will report back when I've had time to do that.

**Reply by jarmitage** [reply]
Hi @gabestein I randomly made the discovery that I can reproduce this issue in the specific case of editing image parameters (e.g. width) on iOS Safari. When I finish editing the parameter the page has scrolled to the top. This doesn't appear to be the case when e.g. setting text styles.
